components/camel-platform-http/src/main/java/org/apache/camel/component/platform/http/cookie/CookieConfiguration.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.component.platform.http.cookie;
+
+/**
+ * Attributes that are set when creating Cookies.
+ */
+public class CookieConfiguration {
```

Comment:
You can add `@Configurer` to let camel source generate code to configure this without reflection.

Then you can configure this also via, something ala:

camel.component.platform-http.cookie-configuration.max-age=2000

Reply:
Added the `@Configurer` annotation and changed the properties to `cookiexxx` to prevent duplicate names.  https://github.com/apache/camel/pull/13301/commits/c025fdb0fef8ce7cbfae4536be30e8f9b8be86fb